build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -75,6 +75,89 @@ Below is an example for ads.
 
 [/example][/filter]
 
+### Standalone use outside valid AMP documents (optional)
+
+<!-- TODO: Remove backticks from link when guide is live -->
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide `[Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/)`.
+
+#### Example
+
+The example below demonstrates `amp-__component_name_hyphenated__` component in standalone use.
+
+```
+<head>
+...
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+
+<script async custom-element="amp-bento-component-name" src="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.js"></script>
+...
+</head>
+<amp-bento-component>
+  ...
+</amp-bento-component>
+<button id="element-id">
+  Event Trigger
+</button>
+<script>
+  example of one API usage
+</script>
+```
+
+#### Interactivity and API usage
+
+Bento enabled components in standalone use are highly interactive through their API. In Bento standalone use, the element's API replaces AMP Actions and events and [`amp-bind`](https://amp.dev/documentation/components/amp-bind/?format=websites).
+
+The `amp-__component_name_hyphenated__` component API is accessible by including the following script tag in your document:
+
+```
+await customElements.whenDefined('${name}-component');
+const api = await ${name}.getApi();
```

Comment:
This might be something like `amp-__component_name_hyphenated__-component` and `__component_name_pascalcase__.getApi();` here, but will let @caroqliu verify

Reply:
I think we want to apply the first suggestion (`whenDefined('amp-__component_name_hyphenated__')` note: no `-component`)) and leave the second `${name}.getApi()`. 

I think we want camel case instead of pascal case when we call `getApi()`, but I'm not seeing that that's a valid variable we can use. 

Optional: The same can be applied in the above code snippet (`amp-__component_name_hyphenated__` instead of any `amp-bento-component`)